r/R/rzeppelin/R/rServer.R

```diff
@@ -0,0 +1,214 @@
+rServe <- function(sockets) {
```

Comment:
These are all the same thing --- rscala originally has two-way communication.  We don't need that, we only need one-way.  So I took out the functions that provided communication in the opposite direction.  That's the difference you're seeing.  (There are also some purely mechanical changes, like to the name of the package.)  Since this is only deletions and mechanical changes, not the addition or modification of anything that isn't de minimus, its all still under rscala's BSD license.